packages/botonic-react/src/components/button.jsx

```diff
@@ -36,8 +36,10 @@ export const Button = props => {
     if (props.webview) openWebview(props.webview, props.params)
     else if (props.path) sendPayload(`__PATH_PAYLOAD__${props.path}`)
     else if (props.payload) sendPayload(props.payload)
-    else if (props.url) window.open(props.url)
-    else if (props.onClick) props.onClick()
+    else if (props.url) {
+      window.open(props.url)
+      props.onClick()
```

Comment:
is props.onClick only defined when props.url is defined?

Reply:
no, props onclick is the method you want to call from parent